### PR TITLE
#7 Make it works with UUID

### PR DESCRIPTION
**Pull request by bjet007**

### CODE REVIEW

File: src/mui/field/ReferenceManyField.spec.js [hunk 1]
```diff
@@ -77,4 +77,32 @@ describe('<ReferenceManyField />', () => {
         assert.deepEqual(SingleFieldListElement.at(0).prop('data'), {});
         assert.deepEqual(SingleFieldListElement.at(0).prop('ids'), []);
     });
+
+    it('should support record with string identifier', () => {
```
Comment (djhi) [inline src/mui/field/ReferenceManyField.spec.js]: Can you add a test to ensure it still supports numbers as identifiers ?

Comment (bjet007) [inline src/mui/field/ReferenceManyField.spec.js]: Sure, I'll try to do it tonight or tomorrow morning.

**Comment by bjet007** [comment]
I've added the requested test for number support.

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**